frontend/src/__tests__/cypress/cypress/fixtures/resources/yaml/data_connection.yml

```diff
@@ -0,0 +1,18 @@
+kind: Secret
+apiVersion: v1
+metadata:
+  name: aws-connection-ods-ci-ds-pipelines
```

Comment:
@FedeAlonso Can you rename the file and inner fields without "ods-ci" ?
For example:

- Rename this file to: `aws-connection-ds-pipelines.yaml` (btw, yaml instead of yml)
- Remove "ods-ci" from any yaml field or code

Reply:
I have modifed the extensions here [209f98c](https://github.com/opendatahub-io/odh-dashboard/pull/3028/commits/209f98c9b856407ae07ef9eb2ba953abcd0de054) 
But we need to keep some ods-ci string, as it is literally part of the name of the bucket we use for pipelines